modules/cache-common/src/main/java/org/opensearch/cache/common/tier/TieredSpilloverCacheStatsHolder.java

```diff
@@ -167,4 +169,17 @@ public void decrementItems(List<String> dimensionValues) {
     void setDiskCacheEnabled(boolean diskCacheEnabled) {
         this.diskCacheEnabled = diskCacheEnabled;
     }
+
+    @Override
+    public void removeDimensions(List<String> dimensionValues) {
```

Comment:
This method looks identical to the method it is overriding in the parent class. Why was this added?

Reply:
The only difference is the assertion checking the list of values passed in is `dimensionNames.size() - 1` instead of `dimensionNames.size()`, but the logic is otherwise the same. We could remove this assertion, and replace the overridden method with a comment somewhere, but my thinking was that the explicit check makes it clearer to developers how to use `removeDimensions` for the inheriting class.

Reply:
Happy to remove it if you think a comment is sufficient

Reply:
@andrross, please let me know what you think on this, happy to do either way.